lib/topological_inventory/ansible_tower/targeted_refresh/worker.rb

```diff
@@ -48,17 +54,17 @@ def process_message(message)
           payload = JSON.parse(message.payload) if message.payload.present?
 
           logger.info("Received message #{model}##{method}, #{payload}")
-          TargetedRefresh::Processor.process!(message, payload)
+          TargetedRefresh::Processor.process!(message, payload, metrics)
         rescue JSON::ParserError
           logger.error("#{model}##{method} - Failed to parse payload: #{message.payload}")
-          raise
+          metrics&.record_error
         rescue => err
           tasks_id = if payload
                        ids = payload['params'].to_a.collect { |task| task['task_id'] }
                        ids.compact!
                      end
           logger.error("#{model}##{method} - Task[ id: #{tasks_id.to_a.join(' | id: ')} ] #{err.class.name}\n#{err.message}\n#{err}\n#{err.backtrace.join("\n")}")
-          raise
+          metrics&.record_error
```

Comment:
Should we still `raise` after recording? Not sure why we'd remove it otherwise.

Reply:
I removed it from all operations. I don't know why it was there. It means, if `raise` is present, the pod ends each time there is an error in processing message.